src/pages/task/taskTabs/testsTable/TestsTableCore.tsx

```diff
@@ -272,7 +272,7 @@ const getColumnsTemplate = (
       const hasLobsterLink = isLogkeeperLink(htmlDisplayURL);
       const lobsterLink = hasLobsterLink
         ? getUpdatedLobsterUrl(htmlDisplayURL)
-        : getLobsterTestLogUrl(taskId, execution, id, lineNum);
+        : getLobsterTestLogUrl({ taskId, execution, testId: id, lineNum });
```

Comment:
Does this need the group id as well?

Reply:
Nope, we don't actually need to pass in the groupID to the Lobster test log url. The corresponding lobster page is able to retrieve that information to generate the Build Logs page url by passing the taskID, execution and testId to this rest route: `/rest/v2/tasks/${taskId}/tests?execution=${execution}&test_id=${testId}`